### ISSUE TITLE
Not all permissions granted in rbd RBAC deploy

### ISSUE THREAD

**Issue by bruce2409** [kubernetes-incubator/external-storage#622]
When using the rbac deployment as stock, you get the following error:

"failed to get admin secret from [\"kube-system\"/\"ceph-admin-secret\"]: secrets \"ceph-admin-secret\" is forbidden: User \"system:serviceaccount:default:rbd-provisioner\" cannot get secrets in the namespace \"kube-system\"".

To fix this I had to edit deploy/rbac/clusterrole.yaml to add:

```yaml
  - apiGroups: [""]
    resources: ["secrets"]
    verbs: ["get","list"]
```

**Comment by cofyc** [comment]
hi, @bruce2409 

It's because rbac files on master branch is for latest rbd-provisioner. The latest rbd-provisioner does not require cluster-level secrets permissions.

**Comment by jamstar** [comment]
where is the latest rdb-provisioner available at?  the latest listed in the deployment file is from 8 months ago.

**Comment by wongma7** [comment]
should be fixed, see #766
